src/ngraph/runtime/cpu/mkldnn_emitter.cpp

```diff
@@ -2099,3 +2272,831 @@ size_t MKLDNNEmitter::build_quantized_inner_product_forward(
     m_primitive_deps[ip_index] = {input_data_index, weights_index, result_index};
     return ip_index;
 }
+
+// Translate inputs of \p node to TensorViewWrappers.
+static void translate_node_inputs(const Node* node, std::vector<TensorViewWrapper>& args)
+{
+    NGRAPH_ASSERT(args.empty()) << "Expected output vector";
+
+    for (const descriptor::Input& input : node->get_inputs())
+    {
+        const descriptor::Output& output = input.get_output();
+        std::shared_ptr<descriptor::Tensor> tv = output.get_tensor_ptr();
+        // TODO: We should use m_variable_name_map[tv->get_name()] but this map hasn't been
+        // populated when MKLDNN primitives are built. Neither it was when intrinsics where built
+        // for CommonFunctionCollection.
+        args.emplace_back(TensorViewWrapper(tv, /*m_variable_name_map[*/ tv->get_name() /*]*/));
```

Comment:
As the TODO explains, `m_variable_name_map` is empty when the mkldnn primitives are created. It was also empty when mkldnn primitives were created the first time by `CommonFunctionCollection` so I'm not sure how critical this is. Populating this map before creating mkldnn primitives would require moving more code to the entry point of `cpu_external_function`. Not sure how feasible that is.